### PR TITLE
feat: feature flag rpm is being taken care of

### PR DESCRIPTION
**Pull request by palashgdev**

### CODE REVIEW

File: frontend/src/container/ServiceApplication/Columns/ServiceColumn.ts [hunk 1]
```diff
@@ -56,3 +60,9 @@ export const getColumns = (
 		dynamicColumnOption,
 	});
 };
+
+interface ITableColumnProps {
```
Comment (Rajat-Dabade) [inline frontend/src/container/ServiceApplication/Columns/ServiceColumn.ts:64]: Can we move this to `src/container/ServiceApplication/types.ts`

**Review by Rajat-Dabade — CHANGES_REQUESTED** [review]
Just a small minor refactoring change, everything else LGTM 👍

**Comment by Rajat-Dabade** [comment]
Also, the build pipeline is broken.

**Review by Rajat-Dabade — APPROVED** [review]

**Comment by makeavish** [comment]
In graph use ops/min instead of op/s when PreferRPM feature flag is true
Screenshot 2023-08-10 at 06 47 45

**Pull request closed without merging**